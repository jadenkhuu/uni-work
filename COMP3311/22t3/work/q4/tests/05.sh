psql racing -c "select * from q4('pony') order by horse;"
