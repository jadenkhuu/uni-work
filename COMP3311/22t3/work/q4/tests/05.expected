      horse       | average 
------------------+---------
 Al Capony        |     555
 Alimony Pony     |   63809
 Edgar Allen Pony |   19210
 My Little Pony   |   57952
 Pony Montana     |      66
 Pony Soprano     |    7944
 Pony Tail        |   15789
 Pony of my Ownie |   35928
(8 rows)
